configure.ac

```diff
@@ -526,13 +526,10 @@ else
   # Sphinx exits with code 1 when it prints its usage
   sphinxver=`{ $SPHINX --version 2>&1 || :; } | head -n 3`
 
-  if ! echo "$sphinxver" | grep -q -w -e '^Sphinx' -e '^Usage:'; then
-    AC_MSG_ERROR([Unable to determine Sphinx version])
-
   # Note: Character classes ([...]) need to be double quoted due to autoconf
   # using m4
-  elif ! echo "$sphinxver" | grep -q -E \
-       '^Sphinx([[[:space:]]]+|\(sphinx-build[[1-9]]?\)|v)*[[1-9]]\>'; then
+  if ! echo "$sphinxver" | grep -q -E \
+       '^([[[:space:]]]+|Sphinx|sphinx-build|\(sphinx-build[[1-9]]?\)|v)*[[1-9]]\>'; then
```

Comment:
The new version of the regex will match a single digit, optionally preceded by spaces. Instead of removing `^Sphinx`, it should probably be `^(Sphinx|sphinx-build)`.

Reply:
Fixed.